### PR TITLE
New Tutorial for creating customized component for GVRF

### PR DESCRIPTION
**Pull request by nitosan**
Sample projects for Component tutorial

http://gearvrf.org/tutorials/play_with_component/

### CODE REVIEW

File: tutorials/tutorial_7_component/build.gradle [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    repositories {
+        jcenter()
```
Comment (liaxim) [inline tutorials/tutorial_7_component/build.gradle]: You need to the google repository too. ``google()``

**Comment by liaxim** [comment]
Looks good to me. The cube could use a different texture. @NolaDonato Any comments?

**Comment by NolaDonato** [comment]
This tutorial is great. I think we should do a javascript version too using GVRScriptBehavior.

**Comment by liaxim** [comment]
Ok, planning to merge on Wednesday. Btw, without adding the google maven repo it doesn't build for me.

**Comment by nitosan** [comment]
added `google()` maven repo, tested locally, let me know if there is any other issue.

**Pull request merged**